app/code/community/Ebizmarts/MailChimp/controllers/Adminhtml/EcommerceController.php

```diff
@@ -87,8 +87,7 @@ public function createMergeFieldsAction()
         $subEnabled = $helper->isSubscriptionEnabled($scopeId, $scope);
         if ($subEnabled) {
             try {
```

Comment:
This try & catch should be removed to allow the createMergeFields catch the errors and give the success value correctly.